pages/open-positions.md

```diff
@@ -5,6 +5,15 @@ redirect_from: /roles-and-teams/
 styles: assets/css/leverpostings.css
 scripts: assets/js/leverpostings.js
 ---
+
+###Important Announcement###
```

Comment:
Not set on this format. Just wanted to capture the need to call attention to this information.